backend/app/controllers/concerns/spree/admin/breadcrumbs.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Spree
+  module Admin
+    module Breadcrumbs
+      def add_breadcrumb(name, path = nil)
+        @admin_breadcrumbs ||= []
```

Comment:
Can we make this a `Set`, so that we do not get potential duplicates and gain some performance?